### ISSUE TITLE
Sidebar with "show previews of messages": make it clear if last message was from yourself

### ISSUE THREAD

**Issue by axelsimon** [vector-im/element-web#25418]
#### What would you like to do?
I would like to know at a glance if i was the last person to send a message in a chat / room or not.

#### Why would you like to do it?
To know quickly if i'm waiting on someone to answer me, or i should be the one sending the next message.

#### How would you like to achieve it?
When the sidebar has "show previews of messages" enabled, it would be great to simply prepend "**me:**" in front of the message preview if it was from me, and nothing (the current situation) if it was from someone else.

### Have you considered any alternatives?

An alternative is to always put the name of the last person who spoke in the room. This opens up a lot of complexity, as users may or may not have set a friendly display name, have funky characters in their name, etc.

"**me:**" keep it simple while solving the issue

**Comment by daniellekirkwood** [comment]
@VolkerJunginger & @amshakal  We've been talking about Web previews and this is another idea for you to consider... Lets chat about it along with the other PR for previews and see if we can move it forwards

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-meta/issues/1603
